packages/cloud-cognitive/src/components/AddSelect/_carbon-imports.scss

```diff
@@ -7,4 +7,13 @@
 
 // AddSelect uses the following Carbon components:
 // Button, OverflowMenu, OverflowMenuItem
+@use '@carbon/react/scss/components/accordion';
```

Comment:
This will work fine, but I just noticed that the styles for the structured list are imported from `@carbon/styles` but all the new carbon imports are from `@carbon/react`. We should probably stick to one or the other for consistency